src/prebid.js

```diff
@@ -342,11 +342,11 @@ $$PREBID_GLOBAL$$.clearAuction = function() {
 };
 
 /**
- *
- * @param bidsBackHandler
- * @param timeout
- * @param adUnits
- * @param adUnitCodes
+ * @param {Object} defaults
```

Comment:
should we call this something else? Like `requestOptions` or something?

Reply:
Good point - changed to `requestOptions`.